tox.ini

```diff
@@ -1,14 +1,43 @@
 [tox]
-envlist = py26,py27,py34,py35
+envlist =
+    py{26,27,34,35}-pyzmq{,13,14,15}
+    py{27,35}-pyzmq13.{0,1}
+    py{27,35}-pyzmq14.{0,1,2,3,4,5,6,7}
+    py{27,35}-pyzmq15.{0,1,2,3,4}
+    flakes
 
 [testenv]
 deps =
-    flake8
     nose
-commands =
-    flake8 zerorpc bin
-    nosetests -v
-passenv = ZPC_TEST_TIME_FACTOR
+
```

Comment:
because of the requirements?

if I were to permutate all sub versions of pyzmq I would not be able to set the requirement for each one

if you are aware of a better way to right this please submit the changes. As a maintainer of zerorpc-python you have access to `medecau:update-tests`